sklearn/tree/export.py

```diff
@@ -380,6 +381,12 @@ def recurse(tree, node_id, criterion, parent=None, depth=0):
     own_file = False
     return_string = False
     try:
+        if decision_tree.tree_ is None:
```

Comment:
You can import `check_is_fitted` from `sklearn.utils.validation` and then do `check_is_fitted(decision_tree, tree_)`